src/OSPSuite.Presentation/Presenters/PresenterConstants.cs

```diff
@@ -13,6 +13,7 @@ public static class PresenterKeys
          public static readonly string ParameterIdentificationResidualVsTimeChartPresenter = createConstant("ParameterIdentificationResidualVsTimeChartPresenter");
          public static readonly string ParameterIdentificationResidualHistogramPresenter = createConstant("ParameterIdentificationResidualHistogramPresenter");
          public static readonly string ParameterIdentificationPredictedVsActualChartPresenter = createConstant("ParameterIdentificationPredictedVsActualChartPresenter");
+         public static readonly string SimulationPredictedVsActualChartPresenter = createConstant("SimulationPredictedVsActualChartPresenter");
```

Comment:
let's call this SimulationPredictedVsObservedChartPresenter?

what about htis one? `ParameterIdentificationPredictedVsActualChartPresenter ` is this being used anywhere?
We should probably rename this as well

Reply:
Yes both are used, probably something outdated. Renamed.